go.mod

```diff
@@ -3,87 +3,99 @@ module github.com/rancher/shepherd
 go 1.20
 
 replace (
-	k8s.io/api => k8s.io/api v0.27.4
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.27.4
-	k8s.io/apimachinery => k8s.io/apimachinery v0.27.4
-	k8s.io/apiserver => k8s.io/apiserver v0.27.4
-	k8s.io/cli-runtime => k8s.io/cli-runtime v0.27.4
-	k8s.io/client-go => github.com/rancher/client-go v1.27.4-rancher1
-	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20230501164219-8b0f38b5fd1f
+	k8s.io/api => k8s.io/api v0.25.12
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.25.12
+	k8s.io/apimachinery => k8s.io/apimachinery v0.25.12
+	k8s.io/apiserver => k8s.io/apiserver v0.25.12
+	k8s.io/cli-runtime => k8s.io/cli-runtime v0.25.12
+	k8s.io/client-go => github.com/rancher/client-go v1.25.4-rancher1
+	k8s.io/kubectl => k8s.io/kubectl v0.25.12
 )
 
 require (
 	github.com/Masterminds/semver/v3 v3.2.0
 	github.com/aws/aws-sdk-go v1.44.322
-	github.com/creasty/defaults v1.7.0
+	github.com/creasty/defaults v1.5.2
 	github.com/gorilla/websocket v1.5.0
-	github.com/hashicorp/go-multierror v1.0.0
+	github.com/hashicorp/go-multierror v1.1.1
 	github.com/pkg/errors v0.9.1
-	github.com/pkg/sftp v1.10.1
-	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.69.1
-	github.com/rancher/apiserver v0.0.0-20230831052300-120e615b17ba
-	github.com/rancher/eks-operator v1.3.0
-	github.com/rancher/fleet/pkg/apis v0.0.0-20231017140638-93432f288e79
-	github.com/rancher/gke-operator v1.2.0
-	github.com/rancher/lasso v0.0.0-20230830164424-d684fdeb6f29
-	github.com/rancher/norman v0.0.0-20230831160711-5de27f66385d
-	github.com/rancher/rancher/pkg/apis v0.0.0-20231025153948-bd93f3227121
-	github.com/rancher/rancher/pkg/client v0.0.0-20231127155507-649110f64f1e
-	github.com/rancher/rke v1.5.0-rc9
-	github.com/rancher/system-upgrade-controller/pkg/apis v0.0.0-20231123175309-a70667b30261
+	github.com/pkg/sftp v1.13.5
+	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.52.0
+	github.com/rancher/apiserver v0.0.0-20230810214006-d134152ce6fb
+	github.com/rancher/eks-operator v1.2.2
+	github.com/rancher/fleet/pkg/apis v0.0.0-20230821130910-23e1c146755a
+	github.com/rancher/gke-operator v1.1.6
+	github.com/rancher/lasso v0.0.0-20230810170912-a86ba6c749aa
```

Comment:
It seems like the previous lasso version `v0.0.0-20230830164424-d684fdeb6f29` on 2.7 r/r